### PR TITLE
Added new student to Students Section

### PR DESCRIPTION
**Pull request by mikolajhojda**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request
New student.
Before:
before
before2
After:
after2
after

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`
- Preview Link: https://mikolajhojda.github.io/gci18.fossasia.org/

## Description / Changes
Added new student to Students Section. Change in _data/students.yml and img/avatar.png

### CODE REVIEW

**Comment by rahulkumaran** [comment]
Hey @mikolajhojda ! 2 things here
Firstly, there's a merge conflict! So try solving it
Secondly, you've checked the box of "squashed the commits" but you still have 8 commits that you want to push. Would be great if you could squash your commits.

File: _data/students.yml [hunk 1]
```diff
@@ -49,7 +53,9 @@
   github: arjunpandey2003
   image: arjun.jpg
 
+
```
Comment (jogendra) [inline _data/students.yml]: Remove extra line!

File: _data/students.yml [hunk 2]
```diff
@@ -49,7 +53,9 @@
...
 - name: Jeewal Kunwar 
   github: Jeewal
   image: JK.jpg
   twitter: jeewalkunwar1
+
```
Comment (jogendra) [inline _data/students.yml]: Same here

**Comment by mikolajhojda** [comment]
Everything is good now?

**Pull request merged**